test/OrchardCore.Tests/Apis/Context/AuthenticationContext.cs

```diff
@@ -17,14 +15,15 @@ internal class PermissionContextAuthorizationHandler : AuthorizationHandler<Perm
 
         // Used for http based graphql tests, marries a permission context to a request
         public PermissionContextAuthorizationHandler(
-            IServiceProvider serviceProvider,
             IHttpContextAccessor httpContextAccessor,
             IDictionary<string, PermissionsContext> permissionsContexts)
         {
             _permissionsContext = new PermissionsContext();
 
-            var httpContext = serviceProvider.GetRequiredService<IHttpContextAccessor>().HttpContext;
-            var requestContext = httpContext != null ? httpContext.Request : httpContextAccessor.HttpContext.Request;
+            if (httpContextAccessor.HttpContext == null)
+                return;
```

Comment:
In OC this is the convention:

```suggestion
            if (httpContextAccessor.HttpContext == null)
            {
                return;
            }
```